src/containers/Locks.js

```diff
@@ -145,12 +195,14 @@ class Locks extends Component {
   renderButtons() {
     const result= {}
     const {policies} = this.props
+		console.log(policies);
```

Comment:
Perhaps we should remove console.log statements

Reply:
You are right, I will remove it.